entrypoints/content/components/secondary/comment/CommentAction.tsx

```diff
@@ -0,0 +1,71 @@
+
+import { useState } from 'react'
+
+
+// Components::
+import { ArrowBigDown, ArrowBigUp, Ellipsis, Forward, MessageSquare } from 'lucide-react'
+
+// Types:
+import { VoteCount } from 'types/votes'
+
+interface VoteCountProps {
+  voteCount: VoteCount
+}
+
+const CommentAction: React.FC<VoteCountProps> = ({ voteCount: { up = 0, down = 0 } }) => {
```

Comment:
also, as a general comment, i think i should be adding the code to hook this up to the backend i.e. implementing the upvote, downvote function calls. right now you have 0 reference and it will be painful for you to write from scratch + for me to refactor it. instead, once the ui issues are fixed, ill add the two functions for that, and then you can take those as reference and add the other functions (there's going to be bookmarkComment, reportComment, blurring the comment based on user preferences and comment's isNSFW, etc.)